x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/definition.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { entityDefinitionSchema, type EntityDefinition } from '@kbn/entities-schema';
+
+export const HOST_ENTITY_DEFINITION: EntityDefinition = entityDefinitionSchema.parse({
+  id: 'ea_host_entity_store',
+  name: 'EA Host Store',
+  type: 'node',
+  indexPatterns: ['log*'],
+  identityFields: ['host.name'],
+  displayNameTemplate: '{{host.name}}',
+  metadata: [
+    'host.domain',
+    'host.hostname',
+    'host.id',
+    'host.ip',
+    'host.mac',
+    'host.name',
+    'host.type',
+    'host.architecture',
+  ],
+  history: {
+    timestampField: '@timestamp',
+    interval: '1m',
+  },
+  version: '1.0.0',
+});
+
+export const USER_ENTITY_DEFINITION: EntityDefinition = entityDefinitionSchema.parse({
+  id: 'ea_user_entity_store',
+  name: 'EA User Store',
+  type: 'node',
```

Comment:
What are the possible values for `type`? There must be something like `user`. 🙏

Reply:
I dont think so. I'm not sure what's the latest on what this field is used for, but I think it simply shows up as `entity.type`. Since it's just a string anyways, I';; update to both user and host